.github/workflows/pub.yml

```diff
@@ -10,13 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-
       - run: pip3 install --user -r requirements.txt
-      - run: cd cn && python3 -m mkdocs build
-      - run: cd en && python3 -m mkdocs build
-      - run: mv cn/site _site
-      - run: mv en/site _site/en
-      - run: cp CNAME _site/
```

Comment:
CNAME 其实与版本无关，没必要每次都拷贝。可以切换到 gh-pages 分支手动上传一次。这也是 mike 官网给的建议：

> Some special files that you'd like to deploy along with your documentation (such as CNAME) aren't related to a particular version of the docs, and instead need to go in the root directory of your site. There's no special handling for this in mike, but since your built docs live on a git branch, it's still easy to manage: check out your gh-pages branch (or wherever your built docs live), and commit the necessary files to the root directory.

Reply:
这里还是有区别的，这个文档假设的是gh-pages分支是自己手动维护的。我们现在的情况不一样，GitHub actions运行的时候相当于会把gh-pages分支的根目录的内容替换成我们产生的'_site'目录下所有的内容。我之前也没加拷贝的这一步，这个文件就没了

Reply:
mike deploy的行为相当于对gh-pages分支不停增加新的文件，我们的使用方式是每次都产生gh pages分支最新版本的所有文件。

Reply:
已经修改